shell.nix

```diff
@@ -16,7 +16,12 @@ craneLib.devShell
       packages = with pkgs; with binPkgs; [
         cargo-dist
         cargo-flamegraph
+        # https://crane.dev/API.html?highlight=mold#cranelibcargonextest
+        # cargo-nextest
```

Comment:
I saw, either somewhere in this PR or in #872, that you're avoiding this for the time being because some parallelism doesn't play nice with `cargo-nextest`...